### ISSUE TITLE
Content and graphics

### ISSUE THREAD

**Issue by Girlwithasmile** [Girlwithasmile/Unique-Frontends#3]
If you are a content writer or a graphic designer looking forward to contribute.. comment here!

**Comment by Khushi-design-20** [comment]
I can contribute as a designer!

**Comment by Girlwithasmile** [comment]
Yeah.. sure! Pls open an issue with the topic of your graphic design. You can have a look at issue#2

**Comment by Khushi-design-20** [comment]
I can create a landing page design in Figma for those topics, that works, right?

**Comment by Girlwithasmile** [comment]
Yeah... Create a seperate branch other than masters, create a folder titled graphic designs n then do that...
